sdk/textanalytics/ai-text-analytics/test/public/utils/env.browser.ts

```diff
@@ -1,2 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
nit: could you also delete the mapping for this file from the browser entry in package.json?